### PR TITLE
Fix for breaking changes in Dash.js 2.4.0

### PR DESCRIPTION
**Pull request by tri170391**
There are 2 changes that breaks the wrapper on 2.4.0:
- [Change in xhr.withCredentials interface](https://github.com/Dash-Industry-Forum/dash.js/commit/5960486814405e29155f800609dacc20887a7d8b).
- [Refactoring of event names](https://github.com/Dash-Industry-Forum/dash.js/pull/1582/commits/1b037ce130e8fced49a8f7715eeed2f9ddd8285b).

I attempted to fix these issues and add in 2 pages to test specific version of Dash.js with the build.

### CODE REVIEW

File: lib/FragmentLoaderClassProvider.js [hunk 1]
```diff
@@ -90,10 +90,19 @@ function FragmentLoaderClassProvider(wrapper) {
         }
 
         function _getSRRequest(request, headers) {
+
+            // From dashjs 4.8 getXHRWithCredentials is replaced with getXHRWithCredentialsForType
```
Comment (ruslandinov) [inline lib/FragmentLoaderClassProvider.js]: Typo in dashjs verison

File: lib/FragmentLoaderClassProvider.js [hunk 2]
```diff
@@ -90,10 +90,19 @@ function FragmentLoaderClassProvider(wrapper) {
...
+            let withCredentials;
+            if (typeof mediaPlayerModel.getXHRWithCredentialsForType === 'function') {
```
Comment (ruslandinov) [inline lib/FragmentLoaderClassProvider.js]: Let's add a comment here explaining why we check this, and putting related dash.js versions on each line

**Comment by tri170391** [comment]
It would be trivial with PHP but I am looking for something totally HTML5 so it might take a while...

**Comment by tri170391** [comment]
I think the fixes on Dash.js have higher priority so I will work on your issue on a separate PR. @ruslandinov

**Comment by ruslandinov** [comment]
Just one thing left -- update the CHANGELOG, and we are good here.

**Pull request merged**